acceptance/tests/apply/augeas/services.rb

```diff
@@ -0,0 +1,61 @@
+test_name "Augeas services file" do
+
+  tag 'risk:medium'
+
+  confine :except, :platform => 'windows'
+  confine :except, :platform => 'osx'
```

Comment:
Could I trouble you to add that as a comment, so I don't ask again in the future. :)